src/systems/filecoin_vm/runtime/impl/runtime.go

```diff
@@ -389,9 +387,6 @@ func _invokeMethodInternal(
 	rt._running = true
 	ret, exitCode = _catchRuntimeErrors(func() InvocOutput {
 		methodOutput := actorCode.InvokeMethod(rt, method, params)
-		if rt._actorSubstateUpdated {
-			rt._rtAllocGas(gascost.UpdateActorSubstate)
-		}
```

Comment:
This was intended to partially mitigate the problem you mentioned in the top-level comment, viz., an actor should only be charged once even if the substate is updated multiple times. I agree it doesn't completely solve the problem, but I think it is a better approximation to the true cost than the implied policy if we remove it (as in this PR)?

Reply:
Ok, agreed.